pkg/endpoints/pipeline.go

```diff
@@ -414,12 +610,19 @@ func getGitValues(url string) (gitServer, gitOrg, gitRepo string) {
 			repoURL = strings.TrimPrefix(url, "http://")
 		}
 	}
+
+	// example at this point: github.com/tektoncd/pipeline
+	numSlashes := strings.Count(repoURL, "/")
+	if numSlashes < 2 {
+		return "", "", "", errors.New("Url didn't match the requirements (at least two slashes)")
+	}
 	repoURL = strings.TrimSuffix(repoURL, "/")
```

Comment:
Move the line `repoURL = strings.TrimSuffix(repoURL, "/")` above the line `// example at this point: github.com/tektoncd/pipeline`. If you have the url `github.com/tektoncd/`, then it will pass your test of having at least 2 slashes, but will not have the gitRepo. So, you'll want to trim the suffix `/` first.